### PR TITLE
[#267] debug output leaking into CLI non-interactive calls

### LINKED ISSUE #267: Debug and Informational output leaking into CLI non-interactive calls
When calling the CLI 

````
[zingolib/src/grpc_connector.rs:361] uri.clone() = https://mainnet.lightwalletd.com:9067/
[zingolib/src/grpc_connector.rs:368] &client = CompactTxStreamerClient {
    inner: Grpc {
        inner: BoxCloneService,
    },
}
Lightclient connecting to https://mainnet.lightwalletd.com:9067/
```` is printed regardless of the log policy that the user selected.

### PR DESCRIPTION
**Pull request by pacu**
This checks that the CLI has been invoked in interactive mode before printing informational strings on startup.

Non-Interactive invocations will have not initial output.

Also removes dbg! call dangling when creating a GRPC connection and removes dbg call from GrpcConnecto constructor
Closes #267

### CODE REVIEW

**Review by zancas — APPROVED** [review]

**Comment by zancas** [comment]
Very nice!  Thanks!

File: zingoconfig/src/lib.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@ use std::{
     sync::{Arc, RwLock},
 };
 
-use log::{info, LevelFilter};
```
Comment (zancas) [inline zingoconfig/src/lib.rs:8]: This import is needed, to provide access to the `info!` macro called later in the file.   Other than that this PR look good.

**Review by zancas — CHANGES_REQUESTED** [review]
See the line note in zingoconfig/src/lib.rs

File: zingoconfig/src/lib.rs (hunk 1 above)
Comment (pacu) [reply zingoconfig/src/lib.rs:8]: can this be added conditionally with that macro as well so it's obvious that the import it's needed in such macros?

Comment (pacu) [reply zingoconfig/src/lib.rs:8]: I added a change to address this based on this answer in stackoverflow

https://stackoverflow.com/questions/58167542/how-to-group-imports-in-rust-with-conditional-compilation

**Review by zancas — APPROVED** [review]
I re-added the deleted `info`.   After the checks pass this can be merged.

File: zingoconfig/src/lib.rs (hunk 1 above)
Comment (zancas) [reply zingoconfig/src/lib.rs:8]: Ahh..   just read this.

Shall we explicitly qualify instead of using `use` at all?

**Review by zancas — DISMISSED** [review]
Prefer explicit qualification over a `use` statement, especially in a case where the qualified name is only referred to once.

**Review by zancas — APPROVED** [review]
I rebased to add my signature to the commit.

**Pull request merged**